packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -732,6 +742,131 @@ Link: <https://${yourOktaDomain}/api/v1/groups?after=00g1emaKYZTWRYYRRTSK&limit=
 ]
 ```
 
+#### List Groups with Search
+
+<ApiLifecycle access="ea" />
+
+Searches for groups based on the properties specified in the search parameter
```

Comment:
Nit: `parameter` &rarr; `parameter.`

Reply:
this doesn't require a period as it isn't a complete sentence. Sentence fragments are the standard intro for endpoints, so it's okay to not have a period here.